src/sasctl/_services/folders.py

```diff
@@ -154,21 +154,21 @@ def create_path(cls, folder, description=None):
             Details of newly-created folder
 
         """
-        if isinstance(folder, str) and folder.count("/") >= 1:
-            # Path must include a leading "/"
-            if not folder.startswith("/"):
-                folder = f"/{folder}"
+        folder = str(folder)
+        # Path must include a leading "/"
+        if not folder.startswith("/"):
+            folder = f"/{folder}"
         else:
```

Comment:
The `else` block is no longer needed (actually causes an issue if folder _does_ start with '/')